### PR TITLE
Desktop version: Creating responsive layout

### PR DESCRIPTION
**Pull request by yosefHesham**
**Brief of the purpose of this branch**
--
This branch aims to make the portfolio responsive as much as possible, to be available for both Mobile and Desktop.

---
**Changes:**
- Made the headline responsive for desktop.
- Used grids to view recent works section with a different number of columns depending on the screen size.
- Used grids to view the about me section with a different number of columns depending on the screen size.
- Made the contact form responsive for desktop by changing flex-direction to row.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,197 +1,274 @@
...
+
+    </section>
+   </div> 
+
+    <section class="recent-works" id="works">
+      <h2 class="section-title">My Recent Works</h2>
+      <hr class="horizontal-line" />
+      <article class="card">
+        <div class="project-sample">
+          <h4 class="project-title">Multi-Post Stories Gain+Glory</h4>
```
Comment (ShoiraTa) [inline index.html:70]: - [ ] Overall you have only 5 cards, but based on requirements the heading is followed by 2 rows with 3 projects each (6 projects in total). Please add one more card ☀️
Screenshot 2022-03-24 at 12 14 00

File: index.html [hunk 2]
```diff
@@ -1,197 +1,274 @@
...
+
+      </div>
+      <img class="disabled-button" src="assets/images/Disabled.svg" alt="" />
+
+    </section>
+   </div> 
+
+    <section class="recent-works" id="works">
+      <h2 class="section-title">My Recent Works</h2>
+      <hr class="horizontal-line" />
```
Comment (ShoiraTa) [inline index.html:67]: - [ ] OPTIONAL: I recommend adding background images to make your page look even better: 
- [ ] Please add more space between about me and recent works section.

Your design  | Figma design
------------- | -------------
Screenshot 2022-03-24 at 12 26 57 | Screenshot 2022-03-24 at 12 27 19

File: index.html [hunk 3]
```diff
@@ -1,197 +1,274 @@
...
+  <section class="contact-me" id="contact">
+    <p class="contact-description">
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </p>
+    <form action="https://formspree.io/f/meqnzwlb" method="POST" class="contact-form">
+      <input type="text" name="name" id="name" placeholder="Full name" class="form-field" maxlength="30" required>
+      <input type="email" name="email" id="email" placeholder="Email address" class="form-field" maxlength="30"
+        required>
+      <textarea class="contact-message" maxlength="500" placeholder="Enter text here" required></textarea>
+      <input type="submit" value="Get in touch" id="submit" class="button">
```
Comment (ShoiraTa) [inline index.html]: - [ ] OPTIONAL: I recommend adding background images in contact section to make your page look even better

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 
----
Hi @yosefHesham ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!

woohoo-youre-almost


 ## Required Changes  :recycle: 

_Check the comments under the review._ 

## Optional suggestions
_Check the comments under the review._ 

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢

Hi @yosefHesham,

Your portfolio is looking great! Keep doing great work 💪💪💪

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

LetgoLetsGooooGIF

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**